src/compile/nodes/InlineComponent.ts

```diff
@@ -20,21 +20,12 @@ export default class InlineComponent extends Node {
 	constructor(component: Component, parent, scope, info) {
 		super(component, parent, scope, info);
 
+		component.warn_if_undefined(info, scope);
+
 		component.hasComponents = true;
 
 		this.name = info.name;
 
```

Comment:
I think to keep track of referenced component names, we could just put in `if (this.name !== 'svelte:component') component.expectedProperties.add(this.name);`.